package.json

```diff
@@ -6,6 +6,9 @@
     "@fontsource/montserrat": "^4.1.0",
     "@fontsource/poppins": "^4.1.0",
     "@fontsource/roboto": "^4.1.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/react-fontawesome": "^0.1.14",
```

Comment:
We can avoid adding these dependencies by using `react-icons/fa`. FontAwesome is already installed 

https://react-icons.github.io/react-icons/search?q=spin